core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -382,10 +382,11 @@ private function handleTokenCheckerConfig(ContainerBuilder $container): void
         }
 
         $tokenChecker = $container->getDefinition('contao.security.token_checker');
-        $tokenChecker->replaceArgument(5, new Reference('security.access.simple_role_voter'));
```

Comment:
Should that be part of that  PR?

Reply:
I have moved it to the `else` part trying to fix https://github.com/contao/contao/runs/7323906436?check_suite_focus=true. However, it did not work.

There is no point in replacing the argument twice, though.